salt/orch/kubernetes.sls

```diff
@@ -72,9 +72,9 @@ etcd_proxy_setup:
     - tgt_type: grain_pcre
     - sls:
       - etcd-proxy
-    - concurrent: True
+    - batch: 5
```

Comment:
Does this `batch` imply that provisioning will happen in parallel in these nodes? Otherwise I think this will not work, as all the masters must be started at the same time so they can create the initial cluster...